### DISCUSSION TITLE
GitHub Campus Program

### DISCUSSION THREAD

**Discussion by priyesranjan** [community/community#110542] [GitHub Education]
How to verify government engineering  college in GitHub campus Program????

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @PRIYESRANJAN , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category . I've gone ahead and moved it for you. Good luck!

**Comment by samus-aran** [comment]
Hi @PRIYESRANJAN 

You can ask your institution to join the GitHub Campus Program by [applying here](https://education.github.com/schools). 👈 

Hope that helps 👍

**Reply by priyesranjan** [reply]
Sir I'm Compus Tech Leader but why my application not approved

**Reply by samus-aran** [reply]
Hi @PRIYESRANJAN 

Usually there are rejection reasons provided. We can confirm you are approved as a verified student 👍  May we ask what you are requesting for?

**Comment by priyesranjan** [comment]
Sir I'm College campus leader but github not approved why?????

**Comment by priyesranjan** [comment]
[Gec Banka certificate.pdf](https://github.com/community/community/files/14534409/Gec.Banka.certificate.pdf)
![Tech Community](https://github.com/community/community/assets/46890021/51394e2e-c499-4acc-802e-c26976f91e23)
I applying everything in good manner but my application is Rejected.

?????????????????????????????


Please help me 
How to apply GitHub School Program to approve my application.
